### DISCUSSION TITLE
With a Code Signing sponsor package, will meshcentral also be signed?

### DISCUSSION THREAD

**Discussion by MichelGreijmans** [amidaware/tacticalrmm#1549] [Q&A] [answered]
Hi team,

If we subscribe signing package through a sponsor package, will the meshcentral binaries be signed as well, or just the TacticalRMM binaries? We've had issues in the past with the meshcentral binaries specifically being deleted/blocked by Bitdefender.

**Comment by wh1te909** [comment, marked as answer]
no but we we are considering doing it in the future as we plan to maintain our own meshcentral fork since development on it is stale

**Comment by compwiz21** [comment]
As a developer ourself, we have our own codesigning certificate. How can we use our own?

**Reply by wh1te909** [reply]
contact support and we can set you up a private code signing server https://docs.tacticalrmm.com/support/#commercial-support
